pkg/network/ebpf/c/protocols/classification/protocol-classification.h

```diff
@@ -209,6 +224,8 @@ __maybe_unused static __always_inline void protocol_classifier_entrypoint(struct
 
  next_program:
     classification_next_program(skb, classification_ctx);
```

Comment:
will this fallthrough when the called function returns?

Reply:
removed